packages/extension-base/src/background/handlers/State.ts

```diff
@@ -48,7 +49,8 @@ type Providers = Record<string, {
 
 interface SignRequest {
   account: AccountJson;
-  id: string;
+  id: string,
+  requestingTabId: number,
```

Comment:
semicolon

Reply:
did I mention that the eslint config in this repo is messed up? 🤣